addons/purchase/tests/test_purchase_order_report.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+
+from datetime import datetime
+from odoo.tests import common
+
+
+@common.tagged('post_install','-at_install')
```

Comment:
Why do you need those tags?

Reply:
or you maybe don't need to explicitly call self.env.ref('l10n_generic_coa.configurable_chart_template').load_for_current_company(False, False)
if you are in post_install

Reply:
We need to set CoA for new company because we are creating invoice for new company and its need the accounts.